### PR TITLE
Add dc_burns api docs

### PR DESCRIPTION
**Pull request by madninja**

### CODE REVIEW

File: docs/api/blockchain/dc_burns.mdx [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+---
+
+## DC Burn Totals
+
+```
+GET https://api.helium.io/v1/dc_burns/sum
+```
+
+Returns the total Data Credits (DC) burntfor the whole network in a given
```
Comment (PharkMillups) [inline docs/api/blockchain/dc_burns.mdx]:
```suggestion
Returns the total Data Credits (DC) burned for the whole network in a given
```

File: docs/api/blockchain/dc_burns.mdx [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+timeframe. Timestamps can be given in ISO 8601 format (e.g.
+`2020-08-27T00:00:00Z`) or in relative time (e.g. `-1 week`, which when url
```
Comment (PharkMillups) [inline docs/api/blockchain/dc_burns.mdx]:
```suggestion
`2020-08-27T00:00:00Z`) or in relative time (e.g. `-1 week`, which, when url-
```

File: docs/api/blockchain/dc_burns.mdx [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+esacped becomes `-1%20week`). When ommitted the current time (UTC) is assumed.
```
Comment (PharkMillups) [inline docs/api/blockchain/dc_burns.mdx]:
```suggestion
escaped becomes `-1%20week`). When omitted the current time (UTC) is assumed.
```

File: docs/api/blockchain/dc_burns.mdx [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+
+The results can also be bucketed in time by specifying a `bucket` query
+parameter which buckets information per bucket in the given timeframe. Data is
+bucketed by time. Valid bucket values include `hour`, `day` and `week`).
+
+For example to get the last 7 days of DC burns bucketed by day use the following
+path/parameters: `?min_time=-7%20day&bucket=day`
```
Comment (PharkMillups) [inline docs/api/blockchain/dc_burns.mdx]:
```suggestion
path/parameters: 

`?min_time=-7%20day&bucket=day`
```

File: docs/api/blockchain/dc_burns.mdx [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+
+---
+
+## DC Burn Events
+
+```
+GET https://api.helium.io/v1/dc_burns
+```
+
+List DC burn events. Events include a block, amount, type of burn and the
```
Comment (PharkMillups) [inline docs/api/blockchain/dc_burns.mdx]:
```suggestion
Use this API to list DC burn events. Events include a block, amount, type of burn, and the
```

File: docs/api/blockchain/dc_burns.mdx [hunk 6]
```diff
@@ -0,0 +1,236 @@
...
+address of the account the DC were burned out of. The oracle price that was
+active at the point the DC burn happened is also listed to enable calculating
+how much HNT the burnt DC represent.
```
Comment (PharkMillups) [inline docs/api/blockchain/dc_burns.mdx]:
```suggestion
how much HNT the burned DC represents.
```

**Comment by PharkMillups** [comment]
Suggested a few small grammar fixes. Otherwise looks great.

**Review by kent-williams — APPROVED** [review]

**Review by PharkMillups — APPROVED** [review]

**Review by PharkMillups — APPROVED** [review]

**Pull request merged**